src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,100 @@
+//VARIABLES:
+@import "../variables.scss";
+
+//WATCH BLOCK:
+
+@mixin animations($name, $duration) {
+  transform-origin: bottom;
+  animation-name: $name;
+  animation-duration: $duration;
+  animation-timing-function: linear;
+  animation-iteration-count: infinite;
+}
+
+.watch {
+  position: relative;
+  width: 600px;
+  height: 600px;
+  margin: auto;
+
+  border: 1px solid #3e0e00;
+  border-radius: 50%;
+
+  background-color: lightgrey;
+  background-image: url("../imgs/img.png");
+  background-size: cover;
+
+  transition: 0.3s;
```

Comment:
Better to define which exact properties will be animated, because if you not define it, it will be applied to every property that can be animated